lib/google_analytics/api.rb

```diff
@@ -0,0 +1,28 @@
+require 'rest_client'
+
+module GoogleAnalytics
+  class Api
+    def self.event(category, action, client_id = '555')
```

Comment:
Ah, I knew there was a note I should add... https://developers.google.com/analytics/devguides/collection/protocol/v1/devguide#overview
client_id has a default value of 555 and all examples I found used it, so I followed the herd! :)

It's overwritten by the form anyway, but it's there as a fallback if it fails